.circleci/config.yml

```diff
@@ -67,4 +67,4 @@ workflows:
       - test:
           matrix:
             parameters:
-              version: ["latest", "1.17", "1.16", "1.15", "1.14", "1.13", "1.12", "1.11"]
+              version: ["1.18", "1.17", "1.16"]
```

Comment:
seems like they no longer publish with `:latest` tag:
```
$ docker pull cimg/go
Using default tag: latest
Error response from daemon: manifest for cimg/go:latest not found: manifest unknown: manifest unknown
```
likely due to ambiguity and confusion of using `latest`